cmd/pg/backup_push.go

```diff
@@ -26,6 +26,7 @@ const (
 	deltaFromUserDataFlag     = "delta-from-user-data"
 	deltaFromNameFlag         = "delta-from-name"
 	addUserDataFlag           = "add-user-data"
+	reduceMemoryUsageFlag     = "reduce-memory-usage"
```

Comment:
Actually, we plan to extract the Files and TarFileSets to the separate file in the future. It can be named, for example, `files_metadata.json`.  So I guess we can name this feature like `without-files-metadata`.

Reply:
@usernamedt `without-files-metadata` feels spot on than the vague `reduce-memory-usage`. I'll proceed with it.